cpp/src/dbscan/corepoints/pack.h

```diff
@@ -16,19 +16,20 @@
 
 #pragma once
 
-namespace raft {
```

Comment:
It's strange git picked up a rename here (a test file to a header in the algorithm) even though the old vs new contnts are completely disjoint. Not a problem, but just curious it was a legitimate file move?

Reply:
It's just Git hallucinating. I removed one and created the other, but they are absolutely unrelated.